pkg/cgroup/v1/blockio.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2021 Andy Dalton
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cgroup
+
+import "github.com/adalton/teleport-exercise/pkg/adaptation/os"
+
+const (
+	BlkioThrottleReadBpsDevice  = "blkio.throttle.read_bps_device"
+	BlkioThrottleWriteBpsDevice = "blkio.throttle.write_bps_device"
+)
+
+// blockIo implements the BlockIO cgroup controller
+type blockIo struct {
+	base
+	// TODO: It might be helpful to enable these to be lists so that a single
+	// blocIo object can handle multiple devices
+	readBpsDevice  *string
```

Comment:
Let's also document that the string is composed of "major:minor bps". Alternatively we could use a stronger type that makes this clearer.